robot_skills/src/robot_skills/arms.py

```diff
@@ -81,18 +87,26 @@ def __init__(self, robot_name, tf_listener, side):
             visualization_msgs.msg.Marker, queue_size=10)
 
     def load_param(self, param_name):
```

Comment:
Should we make this a member of the RobotPart class? Similar stuff is used in arms.py

Reply:
This is arms.py ...